### PR TITLE
Feature #186 and Bugfix #189

### PR DESCRIPTION
**Pull request by bmoscon**
- Allow user to update with a chunk_range
- Fix issue #189 which causes rows and chunk counts to be incorrect after a range delete

### CODE REVIEW

**Comment by bmoscon** [comment]
@jamesblackburn @TomTaylorLondon

File: tests/integration/chunkstore/test_chunkstore.py [hunk 1]
```diff
@@ -342,6 +342,8 @@ def test_update(chunkstore_lib):
     chunkstore_lib.write('chunkstore_test', df, chunk_size='D')
     chunkstore_lib.update('chunkstore_test', df2)
     assert_frame_equal(chunkstore_lib.read('chunkstore_test'), equals)
+    assert(chunkstore_lib.get_info('chunkstore_test')['rows'] == len(equals))
+    assert(chunkstore_lib.get_info('chunkstore_test')['rows'] == len(equals))
```
Comment (jamesblackburn) [inline tests/integration/chunkstore/test_chunkstore.py]: Duplicate assertions?

Comment (bmoscon) [inline tests/integration/chunkstore/test_chunkstore.py]: nice catch!

**Comment by jamesblackburn** [comment]
LGTM

**Pull request merged**